src/gui/GuiRunner.cc

```diff
@@ -161,6 +200,14 @@ void GuiRunner::OnPluginAdded(const QString &_objectName)
     return;
   }
 
+  // The call above always returns the same plugin, which is the first plugin
```

Comment:
Ouch, that's broken. Fix in https://github.com/ignitionrobotics/ign-gui/pull/249

Reply:
@chapulina, We can open an issue to fix this in the future when these two PR are in
 - https://github.com/ignitionrobotics/ign-gazebo/pull/904
 - https://github.com/ignitionrobotics/ign-gui/pull/249

or wait for the forward port.

Reply:
Yeah I wouldn't block this PR on this.

But note that the current implementation is calling `Configure` on the same plugin multiple times, which may go against developers' expectations.

Reply:
Addressed in d6d1f8a